packages/employer-api/src/controllers/application.controller.ts

```diff
@@ -161,14 +142,38 @@ export const updateApplication = async (req: any, res: express.Response) => {
         if (!employerApplicationRecord) {
             return res.status(403).send("Forbidden or Not Found")
         }
-        await applicationService.updateApplication(id, null, req.body)
```

Comment:
We do still need this line, since it's what performs the update operation on the DB.